src/server/views/dashboard/lists-items.njk

```diff
@@ -86,15 +86,15 @@
         </div>
       {% endif %}
 
-      {% if bannerToggles.unpublishWarning.display %}
+      {% if bannerToggles.unpublishWarning %}
         <div class="govuk-notification-banner govuk-notification-banner--error" role="region" aria-labelledby="govuk-notification-banner-title" data-module="govuk-notification-banner">
           <div class="govuk-notification-banner__header">
             <h2 class="govuk-notification-banner__title" id="govuk-notification-banner-title">
               Service providers are going to be unpublished
             </h2>
           </div>
           <div class="govuk-notification-banner__content">
-            <strong>{{ bannerToggles.unpublishWarning.countOfListItems }} service {{ "providers are" if bannerToggles.unpublishWarning.countOfListItems > 1 else "provider is" }} going to be unpublished at 1pm on {{ unpublishDate }} as they have not responded to the annual review. <a href="#" class="govuk-link">Check which providers</a> are going to be unpublished.</strong>
+            <strong>{{ bannerToggles.unpublishWarning.countOfListItems }} service {{ "providers are" if bannerToggles.unpublishWarning.countOfListItems > 1 else "provider is" }} going to be unpublished at 1pm on {{ unpublishDate }} as they have not responded to the annual review. <a href="?activity=out_with_provider" class="govuk-link">Check which providers</a> are going to be unpublished.</strong>
```

Comment:
This is cool, didn't know you could just put query params in href 🤩

Reply:
query param and `#` work in the same way essentially. just ends up appending the query or # to the URL.

if you use nested routers, you can also just append URL parameters without getting the full url first. e.g. if the router is mounted on `/foo/bar`, and the current page is `/foo/bar/baz` then href="fubar" should take you to `/foo/bar/fubar` iirc.